client/src/index.jsx

```diff
@@ -27,7 +27,7 @@ class App extends React.Component {
     return (
```

Comment:
it won't let me add a comment on line 24, but you might want to add a .catch block for errors!